src/app/fdctl/run/tiles/fd_metric.c

```diff
@@ -458,20 +123,11 @@ unprivileged_init( fd_topo_t *      topo,
 
   ctx->topo = topo;
 
-  ctx->conn_id = 0;
-  for( ulong i=0; i<MAX_CONNS; i++ ) {
-    ctx->fds[ i ].fd = -1;
-    ctx->fds[ i ].events = POLLIN | POLLOUT;
-  }
-
-  ctx->fds[ MAX_CONNS ].fd = ctx->socket_fd;
-  ctx->fds[ MAX_CONNS ].events = POLLIN | POLLOUT;
-
   ulong scratch_top = FD_SCRATCH_ALLOC_FINI( l, 1UL );
   if( FD_UNLIKELY( scratch_top > (ulong)scratch + scratch_footprint( tile ) ) )
     FD_LOG_ERR(( "scratch overflow %lu %lu %lu", scratch_top - (ulong)scratch - scratch_footprint( tile ), scratch_top, (ulong)scratch + scratch_footprint( tile ) ));
 
-  FD_LOG_NOTICE(( "Prometheus metrics endpoint listening on port %u", tile->metric.prometheus_listen_port ));
+  FD_LOG_WARNING(( "Prometheus metrics endpoint listening at http://0.0.0.0:%u/metrics", tile->metric.prometheus_listen_port ));
```

Comment:
I prefer notice over warning for this, because it's not a bad thing. Was the thought to make sure it was firewalled appropriately?

Reply:
I think it's good to highlight these lines for now, since we have a good amount of boot spam. When I clean that spam up we can downgrade back to notice.